Sources/UI/PaymentMethods/PaymentMethodListViewController.swift

```diff
@@ -34,6 +39,11 @@ public final class PaymentMethodListViewController: UITableViewController {
         tableView.tableFooterView = UIView(frame: .zero)
         tableView.rowHeight = UITableView.automaticDimension
         tableView.register(PaymentMethodListCell.self, forCellReuseIdentifier: "cell")
+        
+        let emptyViewController = PaymentEmptyViewController()
+        tableView.backgroundView = emptyViewController.view
+        tableView.backgroundView?.isHidden = true
+        self.emptyViewController = emptyViewController
```

Comment:
Kennst du die Containment ViewController API?

Ein Beispiel von [HackingWithSwift](https://www.hackingwithswift.com/example-code/uikit/how-to-use-view-controller-containment) finde ich sehr gut.

Ansonsten hast du [hier](https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/ImplementingaContainerViewController.html) die offizielle Dokumentation.

Reply:
https://github.com/snabble/snabble-ios-sdk/pull/311/commits/8b3a7d8d1c827982dd86485f4558fab860846a46